### PR TITLE
Programmatically switch tab panels and Commands

### PR DESCRIPTION
**Pull request by EugeneTeu**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/PanelType.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * represent the type of commandResult that is executed
+ * so to enable the manager to switch to the correct tab
+ */
+public enum PanelType {
+    DEFAULT,
+    CUSTOMER,
+    PHONE,
+    ORDER,
+    SCHEDULE,
+    STATS;
```
Comment (EugeneTeu) [inline src/main/java/seedu/address/logic/commands/PanelType.java]: @uberSaiyan @zhixianggg  I left a DEFAULT class here

File: src/main/java/seedu/address/logic/commands/PanelType.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.commands;
+
+/**
+ * represent the type of commandResult that is executed
+ * so to enable the manager to switch to the correct tab
+ */
+public enum PanelType {
+    DEFAULT,
```
Comment (zhixianggg) [inline src/main/java/seedu/address/logic/commands/PanelType.java]: good use of enum

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 3]
```diff
@@ -191,11 +208,61 @@ private CommandResult executeCommand(String commandText) throws CommandException
...
         }
     }
+
+    /**
+     * checks which panel the command acts on and switches it
+     * @param input type of panel the result works on
+     */
+    private void switchPanel(PanelType input) {
+        if (input.equals(PanelType.CUSTOMER)) {
+            this.showCustomerPanel();
```
Comment (zhixianggg) [inline src/main/java/seedu/address/ui/MainWindow.java]: makes sense

File: src/main/java/seedu/address/logic/commands/ListCustomerCommand.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+import seedu.address.model.Model;
+
+/**
+ * List all customers in the customer book to the user
+ */
+public class ListCustomerCommand extends Command {
+
+    //to be dicussed
+    public static final String COMMAND_WORD = "to be confirmed";
```
Comment (zhixianggg) [inline src/main/java/seedu/address/logic/commands/ListCustomerCommand.java]: list-c?

File: src/main/java/seedu/address/logic/commands/ListOrderCommand.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+import seedu.address.model.Model;
+
+/**
+ * List all customers in the Order book to the user
+ */
+public class ListOrderCommand extends Command {
+
+    //to be discussed
+    public static final String COMMAND_WORD = "to be confirmed";
```
Comment (zhixianggg) [inline src/main/java/seedu/address/logic/commands/ListOrderCommand.java]: list-o?

**Pull request merged**